packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js

```diff
@@ -39,6 +39,10 @@ export default /* @ngInject */ ($stateProvider) => {
           projectId,
         }),
 
+      goToDiscoveryProjectActivationPage: /* @ngInject */ ($state, projectId) =>
```

Comment:
As it is not a `goTo` method, please rename it
```suggestion
      projectActivationPageHref: /* @ngInject */ ($state, projectId) =>
```